controller/src/main/java/org/jboss/as/controller/descriptions/ModelDescriptionConstants.java

```diff
@@ -192,6 +193,7 @@
     public static final String FILTERED_OPERATIONS = "filtered-operations";
     public static final String FIXED_PORT = "fixed-port";
     public static final String FIXED_SOURCE_PORT = "fixed-source-port";
+    public static final String FOLDER = "folder";
```

Comment:
This should be directory, not folder

http://superuser.com/questions/169457/directory-vs-folder